sys/include/net/ng_ipv6/netif.h

```diff
@@ -84,6 +84,15 @@ extern "C" {
  */
 
 /**
+ * @{
+ * @name Flags for the interfaces
```

Comment:
Thanks for the explanation. Maybe another point for the RIOT best programming practice page.